### ISSUE TITLE
incompatible with annoy==1.17.0 ?

### ISSUE THREAD

**Issue by zktuong** [Teichlab/bbknn#44]
I was trying to run bbknn (and also scrublet) and both independently kept crashing my sessions.

I think it's got to do with `annoy==1.17.0` and simply downgrading to `annoy==1.16.3` made everything run again.

**Comment by ktpolanski** [comment]
Yeah, `annoy==1.17.0` does not seem to work well with both of the specified packages in the environments I've used it in. I have a faint recollection of getting it to work in some setting, but the details escape me.

Maybe I should specify this in the setup file, but the memory of it somehow working has me thinking it might be a local thing somehow?

**Comment by Voltarikk** [comment]
> I was trying to run bbknn (and also scrublet) and both independently kept crashing my sessions.
> 
> I think it's got to do with `annoy==1.17.0` and simply downgrading to `annoy==1.16.3` made everything run again.

i want to how to download annoy==1.16.3,thanks for helping!

**Comment by dfc-1931475** [comment]
> I was trying to run bbknn (and also scrublet) and both independently kept crashing my sessions.
> 
> I think it's got to do with `annoy==1.17.0` and simply downgrading to `annoy==1.16.3` made everything run again.

tk u, my god
